src/sentry/search/events/fields.py

```diff
@@ -461,6 +464,15 @@ def resolve_field_list(
     # Only auto aggregate when there's one other so the group by is not unexpectedly changed
     if auto_aggregations and snuba_filter.having and len(aggregations) > 0:
         for agg in snuba_filter.condition_aggregates:
+            if isinstance(agg, str) and agg in requires_project_threshold_config:
+                if not project_threshold_config:
+                    fields.append(PROJECT_THRESHOLD_CONFIG_ALIAS)
+                    function = resolve_field(
+                        PROJECT_THRESHOLD_CONFIG_ALIAS, snuba_filter.params, functions_acl
+                    )
+                    columns.append(function.column)
+                    project_threshold_config = True
```

Comment:
Does it make sense to move this after the auto aggregates are added to avoid duplicate code?